### ISSUE TITLE
Add Comments to backends.py

### ISSUE THREAD

**Issue by marshalmiller** [marshalmiller/linkrot#17]
Add comments to make it more organized and accessible to others.

**Comment by DevPriSha** [comment]
Hi, I would like to take this issue up. Thanks!

**Comment by marshalmiller** [comment]
@vacom13 I need you to comment on this issue in order to assign it to you. Sorry about that.

**Comment by vacom13** [comment]
@marshalmiller Please assign this to me. Sorry I fell very sick and then college. Didn't get time.😅

**Comment by vacom13** [comment]
@marshalmiller Also, I have unassigned me from the other issue. I assume my comments werent enough 😅. But I would like to add whatever I can to this as well

**Comment by marshalmiller** [comment]
@vacom13 Nah. That was a mistake. I merged your comments. They were helpful. And no need to be sorry. I appreciate you taking the time to contribute to this project.

**Comment by vacom13** [comment]
@marshalmiller Hey. I well tried to think of how to document this better but well came out with nothing. Maybe, you could help me or um maybe I should let someone else do it idk. Well I am probably having a hard time documenting it cus well I am not extremely proficient yet.  And I know documenting is well the easiest part of it all but I do end up having issues with it 😅.  Maybe you could suggest how I go about it?
